Commands/ServerCommands.cs

```diff
@@ -0,0 +1,82 @@
+using ProjectM.Network;
+using Unity.Collections;
+using Unity.Entities;
+
+using VampireCommandFramework;
+
+namespace CommunityCommands.Commands;
+public class ServerCommands {
+	public static class Shutdown {
+		[Command("shutdown", shortHand: "quit", description: "Trigger the exit signal & shutdown the server.", adminOnly: true)]
+		public static void Initialize() {
+			UnityEngine.Application.Quit();
+		}
+	}
+
+	public static class Kick {
```

Comment:
Please reference the existing code in this project for examples of how it uses VCF, this Initialize and implementation style is from RPGMods. You should utilize the existing converters and systems for finding players for example.